robot_smach_states/src/robot_smach_states/navigation/find.py

```diff
@@ -94,8 +95,17 @@ class Find(smach.StateMachine):
     Find an entity based on a description. The description designator should
     have resolve type dict and it should contain at least a 'type' field
     """
-    def __init__(self, robot, knowledge, source_entity_designator, description_designator, area_name_designator,
-                 navigation_area_designator, found_entity_designator):
+    def __init__(self, robot, knowledge, source_entity_designator, description_designator, found_entity_designator,
+                 area_name="on_top_of", navigation_area=None):
+        """
+        :param robot: Robot object
+        :param knowledge: Robocup knowledge object. Used to find an object by its properties
+        :param source_entity_designator: Designator resolving to EdEntity to inspect
+        :param description_designator: dict which contains at least a 'type' field
+        :param found_entity_designator: Designator the found entity can be written to
+        :param area_name: str or str Designator describing the area of the source entity to inspect
+        :param navigation_area: str or str Designator describing the area to drive to for the inspect
```

Comment:
```suggestion
        :param navigation_area: str or str Designator describing the area to drive to for the inspect. The default, 'None', will result in a navigateToObserve otherwise a navigateToSymbolic will be used.
```
This assumes that otherwise a navigateToSymbolic is used